### PR TITLE
Add OS X support to msfupdate

### PR DESCRIPTION
**Pull request by wvu-r7**
This is pretty basic, but I needs it.

### CODE REVIEW

File: config/templates/metasploit-framework-wrappers/msfupdate.erb [hunk 1]
```diff
@@ -100,6 +100,16 @@ EOF
   yum install -y metasploit-framework
 }
 
+install_pkg()
+{
+  (
+    cd ~/Downloads
```
Comment (bcook-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: does this leave a file owned as root under ~/Downloads?

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: It does, but the user owns ```~/Downloads```, so they can ```rm -f``` it. I had ```rm -fv``` in this function, but I'm leaving it off for now. Please don't merge yet. I'll add it once I add status messages.

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: Correction: I also had status messages, but I'm leaving them off for now while I nitpick over whitespace.

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: (```rm -fv``` doesn't look good without a prefixed message.)

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: Whatever. I reverted to the previous revision. Pushing now.

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: Ah, I remember another reason why I removed the ```rm```: the ```return``` breaks it. Fixing.

Comment (wvu-r7) [inline config/templates/metasploit-framework-wrappers/msfupdate.erb]: Done.

**Pull request merged**